Blueprints/BlueprintDefinitions/vs2017/AspNetCoreWebAPI/template/test/BlueprintBaseName.1.Tests/ValuesControllerTests.cs

```diff
@@ -2,15 +2,14 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text.Json;
```

Comment:
Same comment as above. Don't change the VS 2017 blueprints.

Reply:
Reverted.